src/extension.ts

```diff
@@ -1941,12 +1941,23 @@ function isLintable(document: vscode.TextDocument): boolean {
     return document.languageId === 'yaml' || document.languageId === 'json' || document.languageId === 'helm';
 }
 
+function linterDisabled(disabledLinters: string[], name: string): boolean {
+    return disabledLinters.some((l) => l === name);
+}
+
 async function kubernetesLint(document: vscode.TextDocument): Promise<void> {
     // Is it a Kubernetes document?
     if (!isLintable(document)) {
         return;
     }
-    const linterPromises = linters.map((l) => l.lint(document));
+    if (config.getDisableLint()) {
```

Comment:
Move this to the top of the function: if linting is globally turned off, there's no point examining the document to see if we are interested in it.

Reply:
done.